--- strophe.pubsub.js (SHA fda82b14d4377af6c353f4bc51048a68812e03b5)
+++ strophe.pubsub.js (current)
@@ -514,7 +514,7 @@ Extend connection object to have plugin name 'pubs
      */
     setAffiliation: function(node, jid, affiliation, call_back) {
         var that = this._connection;
-        var iqid = thiat.getUniqueId("pubsubaffiliations");
+        var iqid = that.getUniqueId("pubsubaffiliations");
 
         var iq = $iq({from:this.jid, to:this.service, type:'set', id:iqid})
           .c('pubsub', {'xmlns':Strophe.NS.PUBSUB_OWNER})
@@ -544,10 +544,10 @@ Extend connection object to have plugin name 'pubs
             entries.push({
                 data: $build("entry", { xmlns:Strophe.NS.ATOM })
                         .children(atom).tree(),
-                attrs:(atom.id ? { id:atom.id } : {}),
+                attrs:(atom.id ? { id:atom.id } : {})
             });
         }
         return this.publish(node, entries, call_back);
-    },
+    }
 
-});
+});
\ No newline at end of file
